packages/docs-ui/README.md

```diff
@@ -75,3 +76,32 @@ import { SideNavigationSection } from '@braid-design-system/docs-ui';
 
 [`HeadingLevel`]: https://seek-oss.github.io/braid-design-system/components/Heading
 [Braid Design System]: https://seek-oss.github.io/braid-design-system/
+
+### `HeaderNavigation`
+
+Layout for the site logo, a `MenuButton` on smaller screens, and an optional theme selector.
+
+```tsx
+import { HeaderNavigation } from '@braid-design-system/docs-ui';
+
+<HeaderNavigation
+  menuOpen={menuOpen}
+  menuClick={handleMenuClick}
+  logo={<Logo />}
+  logoLabel={logoLabel}
+  themeToggle={<ThemeToggle />}
+/>;
+```
+
+#### Props
+
+| props       | value             | description                                                                                                   |
+| ----------- | ----------------- | ------------------------------------------------------------------------------------------------------------- |
+| menuOpen    | `boolean`         | The Menu can either be open or closed. If open, the button will change to a close icon (defaults to `false`). |
+| menuClick   | `() => void`      | An optional callback function to handle events when the menu button is triggered.                             |
+| logo        | `React.ReactNode` | A React component for the logo of your site (which should act as a link to your homepage).                    |
+| logoLabel   | `string`          | An accessibility label for the logo.                                                                          |
+| themeToggle | `React.ReactNode` | An optional React component for a theme selector.                                                             |
+
+[`HeadingLevel`]: https://seek-oss.github.io/braid-design-system/components/Heading
+[Braid Design System]: https://seek-oss.github.io/braid-design-system/
```

Comment:
These refs have been duplicated. Noticing that the heading level one belongs to the `LinkableHeading` component, but has been moved down as new components have been added, while the braid link must be somewhere else in there too.